Runtimes/Core/CMakeLists.txt

```diff
@@ -175,3 +181,7 @@ add_subdirectory(runtime)
 add_subdirectory(stubs)
 add_subdirectory(CommandLineSupport)
 add_subdirectory(core)
+if(SwiftCore_ENABLE_CONCURRENCY)
+  add_subdirectory(Concurrency/InternalShims)
```

Comment:
Since these are the concurrency internal shims, can the `add_subdirectory` go in `Concurrency/CMakeLists.txt`?

Reply:
Sure.